modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,188 @@
+= Manage Clusters using the Cloud API
+:description: Use the Control Plane API to manage your clusters and organization-level resources in Redpanda Cloud.
+:page-cloud: true
+:tag-clusters: api:ROOT:cloud-api.adoc#tag--ClusterService
+:tag-networks: api:ROOT:cloud-api.adoc#tag--NetworkService
+:tag-operations: api:ROOT:cloud-api.adoc#tag--OperationService
+:tag-resource-groups: api:ROOT:cloud-api.adoc#tag--ResourceGroupService
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your organization's cloud environment outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+For the full Cloud API reference documentation, see xref:api:ROOT:cloud-api.adoc[Redpanda Cloud API Reference].
+
+== Control Plane API
+
+The Control Plane API is one central API that allows you to provision clusters, networks, and resource groups.
+
+The Control Plane API consists of the following endpoints:
+
+* pass:a,m[xref:{tag-clusters}[ClusterService\]]
+* pass:a,m[xref:{tag-networks}[NetworkService\]]
+* pass:a,m[xref:{tag-operations}[OperationService\]]
+* pass:a,m[xref:{tag-resource-groups}[ResourceGroupService\]]
+
+== Long-running operations
+
+Some endpoints do not directly return the resource itself, but instead return an operation. The following is an example response of xref:api:ROOT:cloud-api.adoc#post-/v1beta2/networks[Create Network] (`POST /networks`):
+
+[,bash,role=no-copy]
+----
+{
```

Comment:
this looks like v1beta1 response. v1beta2 should be wrapped into `"operation":{}`

Reply:
@birdayz Added in [this commit](https://github.com/redpanda-data/docs/pull/411/commits/c48e0385fec509770cdeeec96fa1cbfb7406f394#diff-f2b07853a0f8a1ec4ccf92da01e34c0ed1fa79032a329b9a01ff000c21a5f77b), please let me know if this isn't correct